### ISSUE TITLE
Copyright IBM?

### ISSUE THREAD

**Issue by ukmoose** [exobeacon/node-red-node-exobeacon#2]
You code appears to be copywrited to IBM in 2015 is this correct?

**Comment by hardillb** [comment]
Yes, because it looks remarkably like it came from here https://github.com/hardillb/node-red-node-physical-web
